### ISSUE TITLE
Attempt to invoke on a null object reference

### ISSUE THREAD

**Issue by Istiakshovon** [AlexZhukovich/CircleImageView#13]
```plain
java.lang.NullPointerException: Attempt to invoke interface method 'void com.alexzh.circleimageview.ItemSelectedListener.onSelected(android.view.View)' on a null object reference
```

I was going to use the library for my application. But, when I had run it on my device it worked fine. But, when I clicked on that imageView then I got above error.

```xml
<com.alexzh.circleimageview.CircleImageView
                android:id="@+id/imgContact"
                android:layout_width="69dp"
                android:layout_height="104dp"
                android:layout_alignParentTop="true"
                android:layout_centerHorizontal="true"
                android:clickable="true"
                android:src="@drawable/contacts"
                app:layout_constraintBottom_toBottomOf="parent"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toTopOf="parent"
                app:view_borderColor="@android:color/darker_gray"
                app:view_borderWidth="4dp"
                app:view_selectedColor="#0000ff"
                app:view_shadowColor="#808080"
                app:view_shadowDx="2dp"
                app:view_shadowDy="0dp"
                app:view_shadowRadius="4dp"
                tools:ignore="SpeakableTextPresentCheck" />
```

I have added my code above.

**Comment by Istiakshovon** [comment]
Duplicate : https://github.com/AlexZhukovich/CircleImageView/issues/11
Close : #13

**Comment by Istiakshovon** [comment]
Closes : #13
